feat: Add docs for `ignoreErrors` & `ignoreTransactions` (#6642)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

</ConfigKey>

<ConfigKey name="ignore-errors" supported={["javascript", "node"]} notSupported={[]}>

> A list of strings or regex patterns that match error messages that should not be sent to Sentry. This expects a list of the messages to be filtered out before being sent to Sentry as either regular expressions or strings. When using strings, they’ll partially match the messages, so if you need to achieve an exact match, use RegExp patterns instead. By default, all errors are sent.

Reviewer: ```suggestion
A list of strings or regex patterns that match error messages that shouldn't be sent to Sentry. Messages that match these strings or regular expressions will be filtered out before they're sent to Sentry. When using strings, partial matches will be filtered out, so if you need to filter by exact match, use regex patterns instead. By default, all errors are sent.
```

</ConfigKey>

<ConfigKey name="ignore-transactions" supported={["javascript", "node"]} notSupported={[]}>

> A list of strings or regex patterns that match transaction names that should not be sent to Sentry. This expects a list of the names to be filtered out before being sent to Sentry as either regular expressions or strings. When using strings, they’ll partially match the name, so if you need to achieve an exact match, use RegExp patterns instead. By default, all transactions are sent.

Reviewer: ```suggestion
A list of strings or regex patterns that match transaction names that shouldn't be sent to Sentry. Transactions that match these strings or regular expressions will be filtered out before they're sent to Sentry. When using strings, partial matches will be filtered out, so if you need to filter by exact match, use regex patterns instead. By default, all transactions are sent.
```
